### ISSUE TITLE
( UserDetails) Dates in Days Only

### ISSUE THREAD

**Issue by vy-storm** [Juby210/Aliucord-plugins#25]
Presently, the only way to see the time for any given datapoint in "days ago" is to have the exact date visible.

I suggest having a toggle between "Exact Date, Days Ago, Show Both" so if someone wishes to only see data in days ago they can.

**Comment by WitherredAway** [comment]
This has been implemented with the last update

**Comment by vy-storm** [comment]
> This has been implemented with the last update

Hey nice! Is that update out yet for you? I don't see it

**Comment by WitherredAway** [comment]
Yeah it's out for me, not sure why you can't see it tho. Is your aliucord up-to-date?

**Comment by vy-storm** [comment]
> Yeah it's out for me, not sure why you can't see it tho. Is your aliucord up-to-date?

Well lets see, I've got v101.3 of discord, BDC67B7 of aliucord, and 1.1.3 of UserDetails

What bout you?

**Comment by WitherredAway** [comment]
Hmm, I have the exact same versions too. Did you check userdetails' settings? maybe try reinstalling the plugin?

**Comment by vy-storm** [comment]
> Hmm, I have the exact same versions too. Did you check userdetails' settings? maybe try reinstalling the plugin?

I just wanna triple check we are on the same page real quick, can you send a screenshot of your UserDetails settings?

**Comment by WitherredAway** [comment]
![Screenshot_2021-11-14-11-28-40-450_com aliucord](https://user-images.githubusercontent.com/81734495/141669552-bd15302a-4460-4d8a-83dd-6e54a4dc9a4a.jpg)
![Screenshot_2021-11-14-11-28-51-327_com aliucord](https://user-images.githubusercontent.com/81734495/141669553-53ba8209-3697-45c0-9683-d8fa7daf3938.jpg)

**Comment by vy-storm** [comment]
> ![Screenshot_2021-11-14-11-28-40-450_com aliucord](https://user-images.githubusercontent.com/81734495/141669552-bd15302a-4460-4d8a-83dd-6e54a4dc9a4a.jpg) ![Screenshot_2021-11-14-11-28-51-327_com aliucord](https://user-images.githubusercontent.com/81734495/141669553-53ba8209-3697-45c0-9683-d8fa7daf3938.jpg)

Appreciated! And right, that's kinda what I was starting to think might be going on here 😅
Please re-read my suggestion love

**Comment by WitherredAway** [comment]
OHH LMFAO my bad I misinterpret your suggestion 😅

**Comment by vy-storm** [comment]
> OHH LMFAO my bad I misinterpret your suggestion 😅

It's all good 😂 you had me excited there for a second that it had already been done! hopefully it comes about at some point soon tho

**Comment by WitherredAway** [comment]
Yeah, that'd be neat
